requirements.txt

```diff
@@ -4,6 +4,7 @@ numpy
 tensorflow
 h5py
 pillow
+keras
```

Comment:
I disagree with this requirement - we'll need to run it in constrained environments (examples that I have faced were aws micro instances or google app engine; and some of the keras libraries are pretty massive). But the main point is that is not actually needed for any of the core functions. Echoing @ilblackdragon 's suggestion from a couple of months ago, instead of adding keras into the requirements, we can use the following pattern.

```
    try:
        import keras
    except BaseException:
        keras = None
    #
    #
    assert keras is not None
```

Reply:
Ill remove keras from requirements